op-deployer/pkg/deployer/bootstrap/dispute_game.go

```diff
@@ -175,16 +198,33 @@ func DisputeGame(ctx context.Context, cfg DisputeGameConfig) error {
 		release = "dev"
 	}
 
+	// We need to etch the VM and PreimageOracle addresses so that they have nonzero code
```

Comment:
We don't need to update this now, but the VM supports forking now so this code is no longer needed.